tests/float/complex1.py

```diff
@@ -72,6 +80,13 @@
 print(float("inf") * (1 + 1j))
 print(float("-inf") * (1 + 1j))
 
+# malformed complex strings
+for test in ("1+2", "1j+2", "1+2j+3", "1+2+3j"):
```

Comment:
We could throw in `"1 + 2j"` for good measure.

Reply:
added